### ISSUE TITLE
[Radio Buttons] unable to navigate or activate radio buttons when keyboard tabbing in Safari

### ISSUE THREAD

**Issue by mollieru** [18F/web-design-standards#589]
*Steps to recreate*
1. Use keyboard tabbing to navigate to 'Radio Buttons'
2. Tabbing takes user to the first radio button, which is already active
3. Can't navigate between or activate the next two radio buttons 

http://screencast-o-matic.com/watch/coQDbpfdSL

*Expected behavior*
Tabbing throughout the 'Checkboxes' - 'enter' should activate a check box

*Tested on*
OS 10.8
Safari 6.2
